root/components/FormRowSelect.js

```diff
@@ -52,7 +54,7 @@ const FormRowSelect = ({
         options={options}
         required={required}
       />
-      {frozen ? <HiddenField field={field} /> : null}
+      {frozen ? <HiddenField field={field} /> : helpers}
```

Comment:
It's a bit non-intuitive to me that you can only show `helpers` when `frozen` is `false` - they seem unrelated (e.g. I can imagine having a ? button explaining why the field is disabled or what it is).

Reply:
fixed in https://github.com/metabrainz/musicbrainz-server/pull/718/commits/558f1358f6ea589e42ac275f592aa6fffd6e09b6